front/src/rendering-engine/components/engine/engine.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useLayoutEffect, useState } from "react";
+import { BuildTree } from "../../schema/BuildTreeSchema";
+
+import GunComponent from "../gun-component/gun-component";
+import { Canvas, RootWrapper } from "./engine.styles";
+
+interface EngineProps {
+    data: BuildTree;
+}
+
+export const Engine = ({ data }: EngineProps) => {
+    const [ratio, setRatio] = useState<number>(0);
+
+    useLayoutEffect(() => {
+        console.log("renderedCount");
```

Comment:
are we still need this console.log?

Reply:
No. This woud've been deleted via the following PR but I can remove it here as well.